kb-importer/src/main/java/org/eclipse/steady/kb/task/ImportVulnerability.java

```diff
@@ -67,9 +68,10 @@ public void execute(
     String vulnId = vuln.getVulnId();
     this.backendConnector = _backendConnector;
 
-    Boolean overwrite = (Boolean) args.get(OVERWRITE_OPTION);
-    if (!overwrite && getBackendConnector().isBugExisting(vulnId)) {
-      log.info("Bug [{}] already exists in backend, analysis will be skipped", vulnId);
+    if (args.containsKey(DELETE) && (boolean) args.get(DELETE)) {
+      backendConnector.deleteBug(vuln.getVulnId());
```

Comment:
I think this won't always work:
If the ImportVulnerability task is executed before the ImportAffectedLibraries one, then the deletion of the Bug will fail in case there exist affectedlibraries linking the bug construct changes (not only of type KAYBEE but also other types not handled in the ImportAffectedLibraries task). I would propose to skip the deletion of the bug, the analysis would go on and a PUT will be done to update the bug. It may still encounter the problem i described but it would work for "standard" cases whereas it would currently fail in case the ImportVulnerability  is executed before the ImportAffectedLibraries  deleting existing entries.

Reply:
Would it make sense to force these operations to be performed in the right order or do the deletion in Import.java instead?